Snowdog_Components/components/Atoms/typography/heading/heading.config.js

```diff
@@ -146,5 +146,29 @@ module.exports = {
         text: 'Sixth level heading - tag and class'
       }
     },
+    {
+      name: 'page-class',
+      context: {
+        tag: 'h1',
+        class: 'heading heading--page',
+        text: 'Page level heading - only class'
+      }
+    },
+    {
+      name: 'page-tag-and-class',
+      context: {
+        tag: 'div',
+        class: 'heading heading--page',
+        text: 'Page level heading - tag and class'
+      }
+    },
+    {
+      name: 'font-secondary-tag-and-class',
```

Comment:
same as above, this class isn't tag related so I think it should be just
```suggestion
      name: 'font-secondary',
```